### DISCUSSION TITLE
DayPicker overlaps hamburger menu (when the latter is open)

### DISCUSSION THREAD

**Discussion by lelekaspb** [gpbl/react-day-picker#1463] [Support]
Somehow, I get the DayPicker component on top of hamburger links when the hamburger menu is open. How can I fix it without positioning it absolutely and giving it a greater z-index? I just want the picker to stay in the normal flow.

**Comment by gpbl** [comment]
Which hamburger menu? 🤔

**Reply by lelekaspb** [reply]
just a hamburger menu that I made without any packages. it has "display: none" by default, and when the burger button is clicked, the mobile menu covers all other content with z-index higher. But the picker is still on top of my open hamburger menu.
So, I was wondering if there is an easy way to "cancel" that without changing cells' stylings ...?)

**Reply by lelekaspb** [reply]
never mind, I changed the styling of the menu :-)
